app/src/main/java/org/sopt/carrot/domain/model/ProductDetail.kt

```diff
@@ -0,0 +1,18 @@
+package org.sopt.carrot.domain.model
+
+data class ProductDetailModel(
```

Comment:
p1: 저번 리뷰와 같은 이유로 Model 명시 제거 부탁드립니다!! 다른 도메인 모델에도 동일하게 적용 부탁드립니다!!🙇🏻‍♂️🙇🏻‍♂️
```suggestion
data class ProductDetail(
```

Reply:
파일이름만 바꾸고 요건 수정안했네요ㅠ 적용했습니다!